app/src/main/res/layout/list_patient_search_no_patients.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView style="@style/Clinic.V2.TextAppearance.Subtitle1Center.Grey1"
```

Comment:
We are following the convention of using textAppearance for styling attributes related to text rendering. Can you do the same here?

Reply:
done